saleor/userprofile/views.py

```diff
@@ -8,15 +8,17 @@
 from django.urls import reverse
 from django.utils.translation import pgettext
 
+from ..core.utils import get_paginator_items
 from .forms import (
     ChangePasswordForm, get_address_form, logout_on_password_change)
 
-
 @login_required
 def details(request):
     password_form = get_or_process_password_form(request)
+    orders  = request.user.orders.prefetch_related('groups__lines')
+    orders_paginated = get_paginator_items(orders, 5, request.GET.get('page'))
```

Comment:
Why 5?

Reply:
@akjanik There is PAGINATE_BY setting in settings.py, which is used across the storefront.

Reply:
I was about to ask in this PR. Didn't know what number to pick up, `settings.PAGINATE_BY` is 16, isn't too large?

Reply:
^ok then